dspace-api/src/main/java/org/dspace/app/launcher/launcher.xsd

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    The contents of this file are subject to the license and copyright
+    detailed in the LICENSE and NOTICE files at the root of the source
+    tree and available online at
+
+    http://www.dspace.org/license/
+
+-->
+<xs:schema attributeFormDefault="unqualified" elementFormDefault="qualified" xmlns:xs="http://www.w3.org/2001/XMLSchema">
```

Comment:
Should the relationship to /dspace/config/launcher.xsd be documented in this file as well?  Is there a way to utilize only one instance of the file?